executor/builder.go

```diff
@@ -853,10 +853,11 @@ func (b *executorBuilder) buildHashJoin(v *plannercore.PhysicalHashJoin) Executo
 	}
 
 	e := &HashJoinExec{
-		baseExecutor: newBaseExecutor(b.ctx, v.Schema(), v.ExplainID(), leftExec, rightExec),
-		concurrency:  v.Concurrency,
-		joinType:     v.JoinType,
-		innerIdx:     v.InnerChildIdx,
+		baseExecutor:    newBaseExecutor(b.ctx, v.Schema(), v.ExplainID(), leftExec, rightExec),
+		joinConcurrency: v.Concurrency,
+		partConcurrency: int(v.Concurrency),
```

Comment:
I'm not sure, but since partition phase will be the most time-consuming phase, I think we'd better keep these concurrency as two individual variables and test it later.